source/examples/vulkan/vk_color_cube/vk_color_cube.h

```diff
@@ -142,13 +142,6 @@ class AMDVulkanDemo : public gpa_example::GpaSampleApp
     /// Default window height.
     const uint32_t kDefaultWindowHeight = 300;
 
-    /// Note that GPA sample IDs are client defined. However, the Vulkan GPA
-    /// extension assigns an ID to each sample (they are not client defined).
-    /// The GPA library manages the mapping between them. These are the former.
-    static constexpr GpaUInt32 kGpaSampleIdCube = 0;
-    static constexpr GpaUInt32 kGpaSampleIdWireframe = 1;
-    static constexpr GpaUInt32 kGpaSampleIdCubeAndWireframe = 2;
-
```

Comment:
Why are these being removed?

To clarify - these should be defined by the user / application. They do not need to be 0, 1, 2, but could be 4, 5, 12 (or even assigned based on the pointer to the unique object being drawn and profiled). As long as they are unique, the actual value does not matter. We prefer to keep these as user-defined variables for clarity and to show their connection between different GPA calls rather than have them as "magic numbers" that show up throughout the code.